src/labels.js

```diff
@@ -69,10 +49,18 @@ class Labels {
     }
   }
 
-  static initMetadata (wallet) {
-    return Metadata.fromMasterHDNode(wallet._metadataHDNode, METADATA_TYPE_LABELS);
+  get readOnly () {
+    return false;
+  }
+
+  syncWallet () {
+    const syncWallet = () => {
+      this._syncWallet(() => Promise.resolve(), (e) => Promise.reject(e));
```

Comment:
There's a couple of reasons why I'd like to keep the `Labels` class around, despite not storing labels in the KV store.

1 - I did all the refactoring work in this class, moving it to the `HDAccount` class would be difficult (moving a few functions that are clearly in the wrong place, is no problem)
2 - The `HDAccount` test suite takes about a minute to run and that class is already very large
3 - `Labels` are quite complex functionality, especially given the way buy-sell depends on them, so I think it deserves it's own class
4 - It makes it very easy to move labels to the KV store in the future, without a merge-conflict hell

Labels actually only tracks the last _labeled_ receive address index, it doesn't track the last _used_ receive index as returned by the server. That is still handled by `HDAccount`, which combines also those two to find the max. I'll double check to make sure that's in order though.